src/dev.ts

```diff
@@ -0,0 +1,16 @@
+import { startDevOidcProvider } from "src";
```

Comment:
i'd move this out of src (into examples folder or so)

then you can make sure it doesn't get packaged into the npm package